qa/L0_long_running_stress/scenarios.py

```diff
@@ -264,6 +264,7 @@ def generate_sequence_data(self, trial, dtype, data_filename):
                     res = float(res)
                 elif dtype == np.int32:
                     # Do nothing
+                    pass
```

Comment:
had to add this or formatter is sad

Reply:
This is a nice update (comment may or may not be needed now). https://github.com/triton-inference-server/server/pull/3611 (@GuanLuo) also tackles this, though this noop may be more efficient/clearer.